build-tools/packages/build-cli/docs/generate.md

```diff
@@ -142,6 +143,29 @@ EXAMPLES
     $ flub generate changeset --all
 ```
 
+## `flub generate manifest`
+
+As all builds are published to build feed, this command picks the latest successful build against the provided branch name and generates dev manifest file.
+
+```
+USAGE
+  $ flub generate manifest -o <value> -p <value> -t <value> -s <value> [-v | --quiet]
+
+FLAGS
+  -o, --organization=<value>  (required) Organization name
+  -p, --project=<value>       (required) Project name
+  -s, --sourceBranch=<value>  (required) Branch name across which the dev release manifest should be generated.
+  -t, --ado_pat=<value>       (required) ADO Personal Access Token
+
+LOGGING FLAGS
+  -v, --verbose  Enable verbose logging.
+  --quiet        Disable all logging.
+
+DESCRIPTION
+  As all builds are published to build feed, this command picks the latest successful build against the provided branch
```

Comment:
I would say something like:

Creates a release report for the most recent build published to an internal ADO feed. It does this by finding the most recent build in ADO produced from a provided branch, and creates a report using that version. The report always uses the "simple" report format.

Reply:
The report always uses the "caret" report format removing the ranges and replacing with singular dev versions.

Reply:
The caret report has caret ranges. The "simple" report has exact versions. For example:

simple:

```json5
{
  "@fluid-example/app-insights-logger": "2.0.0-internal.6.1.1",
  ...
}
```

caret:

```json5
{
  "@fluid-example/app-insights-logger": ">=2.0.0-internal.6.1.1 <2.0.0-internal.7.0.0",
  ...
}
```

The JSON-serialized formats are the same (so are the types), but since you're creating a manifest with exact versions, regardless of what you use as input, it's a "simple" report.

Reply:
Right, but I need caret ranges for the reamning packages: 

simple: 

{
"@fluid-tools/benchmark": "0.48.0",
  "@fluid-tools/build-cli": "0.22.0",
  }
  
  caret: 
  {
"@fluid-tools/benchmark": "^0.48.0",
  "@fluid-tools/build-cli": "^0.22.0",
  }

Reply:
Ahhh, I see... Maybe note this in a comment around where you're reading the caret JSON, so we (I? :smile:) don't forget why.